pages/preprints.py

```diff
@@ -81,7 +89,7 @@ class PreprintSubmitPage(BasePreprintPage):
     authors_save_button = Locator(By.CSS_SELECTOR, '#preprint-form-authors .btn-primary', settings.QUICK_TIMEOUT)
 
     conflict_of_interest = Locator(By.ID, 'coiNo', settings.QUICK_TIMEOUT)
-    coi_save_button = Locator(By.CSS_SELECTOR, '#author-coi-assertion .btn-primary')
+    coi_save_button = Locator(By.CSS_SELECTOR, '.cp-is-open .btn-primary')
```

Comment:
Similar to `public_data_input` above, and possibly solvable through similarly complicated xpath.  Something like `//section[header/div[text()="Conflict of Interest"]]` to select the parent `section`of the button by a sibling's contents.